File: Heat/Models/OutboundCall.vb
Namespace Models
    ''' <summary>
    ''' Rappresenta una chiamata ad un contatto per fissare un appuntamento.
    ''' </summary>
    Public Class OutboundCall
        Property ID As Integer
        Property CallDate As DateTime
        Property ContactID As Integer
        Overridable Property Contact As Contact
        Property ResultID As Integer

        ''' <summary>
        ''' Chi ha eseguito la chiamata
        ''' </summary>
        ''' <returns></returns>
        Property Login As String
    End Class
End Namespace

File: Heat.ConvertedToC#/Models/OutboundCall.cs
using System;
namespace Heat.Models
{
    /// <summary>
    /// Rappresenta una chiamata ad un contatto per fissare un appuntamento.
    /// </summary>
    public class OutboundCall
	{
		public int ID { get; set; }
		public DateTime CallDate { get; set; }
		public int ContactID { get; set; }
		public virtual Contact Contact { get; set; }
		public int ResultID { get; set; }

		/// <summary>
		/// Chi ha eseguito la chiamata
		/// </summary>
		/// <returns></returns>
		public string Login { get; set; }
	}
}
